server/routes/sentry/sentry_routes.py

```
@@ -0,0 +1,486 @@
+                set_rls_context(cursor, conn, user_id, log_prefix="[SENTRY:get_creds]")
+                cursor.execute(
+                    "SELECT user_id FROM user_tokens WHERE org_id = %s AND provider = 'sentry' AND is_active = TRUE AND secret_ref IS NOT NULL LIMIT 1",
+                    (org_id,)
+                )
+                row = cursor.fetchone()
+
+        if row:
+            return get_token_data(row[0], "sentry") or None
+
+        return None
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)